source/session.ts

```diff
@@ -722,10 +724,10 @@ export class Session {
    * @param  {string} schemaId Id of schema model, e.g. `AssetVersion`.
    * @return {Object|null} Schema definition
    */
-  getSchema(schemaId: string): Data | null {
+  getSchema(schemaId: string): Schema | null {
     for (const index in this.schemas) {
-      if (this.schemas[index].id === schemaId) {
-        return this.schemas[index];
+      if (this.schemas[Number(index)].id === schemaId) {
+        return this.schemas[Number(index)];
```

Comment:
consider rewriting this to use this.schemas.find instead

Reply:
Done